modules/common/log/loki.nix

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+{config, ...}: let
+  loki_data_dir = "/var/lib/loki";
+in {
+  systemd.tmpfiles.rules = [
+    "d /var/lib/loki 0777 loki loki - -"
+  ];
+
+  services.loki = {
+    inherit (config.ghaf.logging.client) enable;
+    configuration = {
+      auth_enabled = false;
+      server = {
+        http_listen_port = 3100;
```

Comment:
Please add an option for the port in [modules/common/log/default.nix](https://github.com/tiiuae/ghaf/pull/625/files#diff-251d008edd24f623160e847eb1ced62c07dea3ebf0b710f10554b8e7fa524ff0)

Reply:
Done.